crates/core/tedge_agent/src/operation_logs.rs

```diff
@@ -1,10 +1,22 @@
-use chrono::{SecondsFormat, Utc};
 use plugin_sm::log_file::LogFile;
 use std::cmp::Reverse;
 use std::collections::BinaryHeap;
 use std::path::PathBuf;
+use time::{format_description, OffsetDateTime};
 use tracing::log;
 
+#[derive(Debug, thiserror::Error)]
+pub enum OperationLogsError {
+    #[error(transparent)]
+    FromIo(#[from] std::io::Error),
+
+    #[error(transparent)]
+    FromTimeInvalidFormatDescription(#[from] time::error::InvalidFormatDescription),
```

Comment:
Since we are now using `format_description::well_known::Rfc3339` this becomes useless.